agimus_controller/utils/record_bag.sh

```diff
@@ -0,0 +1,3 @@
+rosbag record \
+    --output-name=ros_topics \
+    /ctrl_mpc_linearized/ocp_solve_time
```

Comment:
add a new line at the end of the file.
For the rosbag I would prefer a roslaunch.
It's kinda made for this stuff the roslaunch...